src/query_builder.js

```diff
@@ -75,6 +80,15 @@ export default class MQEQuery {
     });
   }
 
+  addFunctions(functions) {
+    var query = "";
+    if (functions.length) {
+        var result = _.map(functions, function (fn, index) {
```

Comment:
`_.map()` function usually used for creating new array. If you need to change external variable, like `query` in this case, use `_.forEach()` or native JS `forEach()`:
```
_.forEach(functions, function (fn) {
  query += "|" + fn + " ";
});
return query;
```